app/breaksPlanner.js

```diff
@@ -135,6 +135,11 @@ class BreaksPlanner extends EventEmitter {
     this.nextBreak()
   }
 
+  correctBy (millis) {
+    if (!this.scheduler) return
+    this.scheduler.reload(millis)
```

Comment:
I am not sure if I would correct the timer here, or inside scheduler. But I guess you are right, scheduler should be able to reschedule itself. So maybe rename the method to `reschedule`? Tell me what you think ;]

Reply:
My reasoning was as follows: the scheduler knows about it's start/delay times so it shouldn't leak that information to the breakPlanner but at the same time it should allow to correct time/reschedule itself as a black box.

It's really kind of hard to come up with a decent name here. I don't think `reload` is a good choice, but I didn't want to have `scheduler.reschedule()` thingie, as it's sort of repetitive. Though both of them are not good enough imo, so I'm fine with using `reschedule` if you want? =)